_sass/_notosanscanadianaboriginal.scss

```diff
@@ -0,0 +1,14 @@
+/*
+ * Noto Sans Canadian Aboriginal (Canadian Aboriginal) http://www.google.com/fonts/earlyaccess
+ */
+@font-face {
+  font-family: 'Noto Sans Canadian Aboriginal';
+  font-style: normal;
+  font-weight: 400;
+  font-display: swap;
```

Comment:
@roedoejet instead of adding the `<link>` to every page, I embedded the appropriate CSS in a Sass include, to reduce the number of HTTP requests made during loading. I also added `font-display: swap` [which is the current best practice](https://css-tricks.com/font-display-masses/#most-of-the-time-youll-use-swap).